prql-compiler/tests/integration/main.rs

```diff
@@ -212,3 +214,15 @@ fn replace_booleans(rows: &mut Vec<Row>) {
         }
     }
 }
+
+// MySQL may adds 0s to the end of results of `/` operator
+fn remove_trailing_zeros(rows: &mut Vec<Row>) {
+    let re = Regex::new(r"^(|-)\d+\.\d+0+$").unwrap();
+    for row in rows {
+        for col in row {
+            if re.is_match(col) {
+                *col = Regex::new(r"0+$").unwrap().replace_all(col, "").to_string();
```

Comment:
Looks reasonable as-is. Can it just be a single `\.0+$` regex for simplicity, without an `if re.is_match`?

Reply:
I think it is worth clarifying that only numbers are covered, but I have no strong opinion.